spec/docs/mapping.md

```diff
@@ -28,6 +28,8 @@ may form one or more [RDF triples]() for each iteration.
   rml:predicateObjectMap <#POM1>, <#POM2>, ..., <#POMn> . 
 ```
 
+Note: A [=triples map=] may not have a [=logical source=] if all its [=term maps=] are [=constant-valued expression maps=] or have a blank node as [=term type=] and no [=expression map=].
```

Comment:
This can be written more clearly and formatted as an aside.
```suggestion
<aside class="note>
A [=triples map=] may be defined  without a [=logical source=] it all its term maps produce values irrespective of a logical iteration. This is the case if all its [=term maps=] are [=constant-valued expression maps=] or have a blank node as [=term type=] and have no specified [=expression=].
```

We probably need a new term and definition for term maps that have a blank node as [=term type=] and have no specified [=expression=].